Update readme (#1676)
Shopify/quilt: README.md

These libraries compose together to help you create performant modern JS apps that you love to develop and test. These packages are developed primarily to be used on top of the stack we like best for our JS apps; Typescript for the flavor, Koa for the server, React for UI, Apollo for data fetching, and Jest for tests. That said, you can mix and match as you like.

> ⚠️ Over the past few years, this repo has evolved into a dumping ground for a variety of packages unrelated to the core problems Quilt aims to solve. For this reason, we strongly discourage the addition of new packages without consulting the Web Foundations team.

Reviewer: ```suggestion
⚠️ Over the past few years, this repo has become a dumping ground for a variety of packages unrelated to the core problems Quilt aims to solve. For this reason, we are no longer accepting new packages.
```

This might be slightly more harsh, but I believe our message should reflect our stance. We don't want more packages, and hosting them in a separate repo will almost always be our answer.
